docs/beyond_basics/101-181-metalad.rst

```diff
@@ -4,10 +4,276 @@ Metadata-Management with MetaLad
 --------------------------------
 
 For many years, :term:`metadata` related functionality was included in the DataLad core package.
-A modernized approach, however, is developed in the `datalad-metalad extension <http://docs.datalad.org/projects/metalad/en/latest/>`_.
+A modernized approach, however, is now developed in the `datalad-metalad extension <http://docs.datalad.org/projects/metalad/en/latest/>`_.
```

Comment:
Should we rather point to the github repository here? At least while the documentation is still outdated.

```suggestion
A modernized approach, however, is now developed in the `datalad-metalad extension <https://github.com/datalad/datalad-metalad>`_.
```

Reply:
yeah, I guess its better than confusing people :shrug: thx for the suggestion